src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -65,45 +68,62 @@
 @PrepareForTest({Main.class, CommonUtil.class})
 public class MainTest {
 
+    private static final String SHORT_USAGE = String.format(Locale.ROOT,
+            "Usage: checkstyle [OPTIONS]... FILES...%n"
+            + "Try 'checkstyle --help' for more information.%n");
+
     private static final String USAGE = String.format(Locale.ROOT,
-          "usage: java com.puppycrawl.tools.checkstyle.Main [options] -c <config.xml>"
```

Comment:
@romani Do we need to create a section documenting how some arguments changed between the versions?
See my next comment on why I am asking.

@remkop Some other applications I have created use the CLI as an api to execute checkstyle.
I know some parameters changed, but I don't know the extent. If any of these are affected, please let me know.
https://github.com/checkstyle/contribution/blob/master/checkstyle-web-tester/checkstyle.php#L136
https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/launch_diff.sh#L208
https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/launch_diff_antlr.sh#L180
https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/sevntu_launch_diff.sh#L208

Reply:
> Do we need to create a section documenting how some arguments changed between the versions?

we can make comment in issue, to reference to it if somebody ask a questions. As far as know we change only "-gxs" (removed) and "-V" for version, all other stay the same. Minor change.

Reply:
I added this comment that summarizes the option name changes: https://github.com/checkstyle/checkstyle/issues/6068#issuecomment-434097233

Reply:
@rnveach I checked the 4 links above and I believe none of them should be impacted by the changes in this PR.

Reply:
@remkop Thanks.